lib/govuk_publishing_components/presenters/machine_readable/government_service_schema.rb

```diff
@@ -0,0 +1,73 @@
+module GovukPublishingComponents
+  module Presenters
+    class GovernmentServiceSchema
+      attr_reader :page
+
+      def initialize(page)
+        @page = page
+      end
+
+      def structured_data
+        # http://schema.org/GovernmentService
+        {
+          "@context" => "http://schema.org",
+          "@type" => "GovernmentService",
+          "name" => page.title,
+          "serviceType" => page.description,
+          "description" => description,
+          "url" => page.canonical_url
+        }
+          .merge(provider)
+          .merge(related_services)
+      end
+
+    private
+
+      def related_services
+        related_links = page.content_item.dig('links', 'ordered_related_items')
+
+        return {} unless related_links.present?
+
+        {
+          "isRelatedTo" => related_links.each_with_object([]) do |link, items|
+            if link['schema_name'] == 'transaction'
+              items << {
+                "@type" => "GovernmentService",
+                "name" => link['title'],
+                "url" => link['web_url']
+              }
+            end
+          end
+        }
+      end
+
+      def provider
+        organisations = page.content_item.dig('links', 'organisations')
+
+        return {} unless organisations.present?
+
+        organisation = organisations.first
```

Comment:
could we return an array of orgs here?  Something like:

```
orgs = organisations.map do |org|{
  {
    "@type" => "GovernmentOrganization",
    "name" => org['title'],
    "url" => org['web_url']
   }
end

{ "provider" => orgs }
```

Reply:
`provider` as far as I can tell isn't an array: https://schema.org/provider